### ISSUE TITLE
Update readme?

### ISSUE THREAD

**Issue by ekanna** [goji/goji#24]
Hi @zenazn,

As **context** pkg landed in standard library of go1.7 tip, you may need to revisit stability section of README file before it wide spread!

Just curious, any specific advantage for changing syntax in new goji

from simple:
``` go
goji.Get("/hello/:name", hello)
```
to verbose pat.Get(...):
``` go
mux.HandleFuncC(pat.Get("/hello/:name"), hello)
```

Some how, i got addicted to your old syntax! Thanks for the great library.

**Comment by infogulch** [comment]
It looks like this was fixed in e355964. I'm also interested in the reasoning behind the redesign!

**Comment by ekanna** [comment]
I made a wrong comparison. If i compare with std lid it make sense. 
``` go
http.HandleFunc("/hello", hello)
mux.HandleFunc(pat.Get("/hello/:name"), hello)
```
Very close to std lib. This is one of the reason why i choose goji over libs like https://github.com/labstack/echo

Thanks
